pkg/osquery/table/platform_tables_darwin.go

```diff
@@ -85,6 +86,7 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 		TouchIDUserConfig(client, logger),
 		TouchIDSystemConfig(client, logger),
 		UserAvatar(logger),
+		falcon_kernel_check.TablePlugin(logger),
```

Comment:
#882 mentions only Linux support - but CrowdStrike supports macOS/Linux? lmk if I should remove